diesel_cli/src/cli.rs

```diff
@@ -57,6 +57,9 @@ pub fn build_cli() -> App<'static, 'static> {
         ).subcommand(SubCommand::with_name("drop")
             .about("Drops the database specified in your DATABASE_URL.")
             .setting(AppSettings::Hidden)
+        ).subcommand(SubCommand::with_name("touch")
```

Comment:
The examples should be dropping and recreating their databases, since "create a migration and run it" is part of the example. The test suite however should not leave behind any state. I don't think that requiring a single command to be run to create the database before running the test suites is a major issue, and it means that most of the test suites do not need to assume that they are running as a user with elevated permissions.